### PR TITLE
Update gallery with thumbnails

### PR DESCRIPTION
**Pull request by alizehrehman**
- Added ThumbnailList Component
- Rendered Thumbnails over the defaultView Component

Note: Still need to make the component stateful to keep track of selected photo and arrows for scroll

### CODE REVIEW

**Comment by alizehrehman** [comment]
@jabrake @matthewwrobel Can you please review this PR? Thank you!

File: client/product_details/DefaultView.jsx [hunk 1]
```diff
@@ -1,18 +1,30 @@
 import React from 'react';
+import ThumbnailList from './ThumbnailList.jsx';
```
Comment (jabrake) [inline client/product_details/DefaultView.jsx:2]: Do you need this import statement in this file? Doesn't look like you're rendering this component here but maybe you will in the future - just checking!

**Review by jabrake — APPROVED** [review]
I reviewed your ThumbnailList and defaultView components (assuming those are the only ones that need review per your comment - let me know if I missed something though!) and it looks good to me. It is showing there's a merge conflict though.

Otherwise, mergey merge!

File: client/product_details/DefaultView.jsx (hunk 1 above)
Comment (alizehrehman) [reply client/product_details/DefaultView.jsx:2]: I forgot that in there. I'm not rendering it in this component anymore. I will update this on the next PR.

**Comment by alizehrehman** [comment]
Merge conflict, resolved! Thanks @jabrake!

**Pull request merged**